spec/graphql/anycable_spec.rb

```diff
@@ -124,7 +122,7 @@
         )
       end
 
-      let(:redis) { AnycableSchema.subscriptions.redis }
+      let(:redis) { AnycableSchema.subscriptions.with_redis { _1 } }
```

Comment:
Is it different from other `let`s on purpose or just leftover?
```suggestion
      let(:redis) { $redis }
```